xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_support.xsd

```diff
@@ -497,4 +497,213 @@ Rail transport, Roads and Road transport
+		</xsd:annotation>
+		<xsd:restriction base="xsd:NMTOKEN">
+			<xsd:enumeration value="none"/>
+			<xsd:enumeration value="carParking"/>
+			<xsd:enumeration value="vegetation"/>
+			<xsd:enumeration value="building"/>
+			<xsd:enumeration value="streetFurniture"/>
+			<xsd:enumeration value="other"/>
+		</xsd:restriction>
+	</xsd:simpleType>
+	<!-- ======================================================================= -->
+	<xsd:simpleType name="BollardCrossingEnumeration">
+		<xsd:annotation>
+			<xsd:documentation>Allowed values for the bollards on crossings</xsd:documentation>
+		</xsd:annotation>
+		<xsd:restriction base="xsd:NMTOKEN">
+			<xsd:enumeration value="none"/>
+			<xsd:enumeration value="contrasted">
+				<xsd:annotation>
+					<xsd:documentation>Bollards on both sides and well contrasted with the environment.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="yes">
+				<xsd:annotation>
+					<xsd:documentation>Bollards on either side of the crossing (with no information about the contrast).</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="partial">
+				<xsd:annotation>
+					<xsd:documentation>Bollards only on one side of the crossing.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="limited">
+				<xsd:annotation>
+					<xsd:documentation>Bollards present but too low or poorly positioned in relation to the crossing.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+		</xsd:restriction>
+	</xsd:simpleType>
+	<!-- ======================================================================= -->
+	<xsd:simpleType name="EntranceTurningSpacePositionEnumeration">
+		<xsd:annotation>
+			<xsd:documentation>Allowed values for the positions of a turning space for an Entrance.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:restriction base="xsd:NMTOKEN">
+			<xsd:enumeration value="none"/>
+			<xsd:enumeration value="outside"/>
+			<xsd:enumeration value="inside"/>
+			<xsd:enumeration value="insideAndOutside"/>
+		</xsd:restriction>
+	</xsd:simpleType>
+	<!-- ======================================================================= -->
+	<xsd:simpleType name="DoorHandleEnumeration">
+		<xsd:annotation>
+			<xsd:documentation>Allowed values for the type of handle for a door</xsd:documentation>
+		</xsd:annotation>
+		<xsd:restriction base="xsd:NMTOKEN">
+			<xsd:enumeration value="none"/>
+			<xsd:enumeration value="lever">
+				<xsd:annotation>
+					<xsd:documentation>Lever handle</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="button"/>
+			<xsd:enumeration value="knob"/>
+			<xsd:enumeration value="crashBar">
+				<xsd:annotation>
+					<xsd:documentation>Panic bar</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="doorHandle">
```

Comment:
So "poignée palière" would be a *doorHandle* and the "pull handle" (second picture) would be a *grabRail*. @Aurige Is that what you need?

Reply:
Tim said those are the correct terms...